_layouts/post.html

```diff
@@ -1,29 +1,24 @@
 ---
 slug: post
 widgets:
-  - twitter
+  - addthis
 ---
 {% include section/header.html %}
-{% capture headline %}<h1>{{ page.title }}</h1>{% endcapture %}
-{% include component/masthead.html headline=headline %}
 
-<div class="container text-line-height-2x py-3">
-  <div class="authors text-center">
-    {% for author in page.authors %}
-    <a href="{{ author.url }}" class="mb-3 badge border badge-light shadow-sm">
-      <img class="rounded" src="{{ author.photo }}" alt="{{ author.name }}" width="32">
-      {{ author.name }}
-    </a>
-    {% endfor %}
-  </div>
+{% if page.options contains 'full-bleed-cover' %}
+<div class="cover-image" style="background-image: url({{ page.cover }});"></div>
+{% endif %}
+
+<div class="container text-line-height-2x pt-3">
+
+  <h1 class="mb-3">
+    {{ page.title }}
+  </h1>
 
   {{ content }}
 
-  <div class="text-center mb-1">
-    <a class="twitter-follow-button" href="https://twitter.com/GodaddyOSS" data-size="large">
-      Follow @GodaddyOSS
-    </a>
-  </div>
+  {% include component/social-share.html %}
+  {% include component/author-panel.html %}
```

Comment:
what's the reasoning for moving the authors and the share buttons to the bottom?

Reply:
It's subjective. I wanted the blog content to be front and center, without any distractions, because that is the meat of what we are giving to visitors. Social share links and links to the authors are of tertiary importance — I can't expect a visitor to share an article they haven't read — and prefer the user engage with the post before leaving to learn more about the authors. 

This also more resembles the pattern used by Medium, except they have a small feature for the author at the top, which I haven't (yet) found a clean solution for since we support multiple authors.

Reply:
I wonder wether there are case studies on things like where should you put your social share buttons to have a bigger impact? I am not that worried about authors being in the bottom, but more for the share buttons

Reply:
a quick search showed there are resource for this:

* https://www.sharethis.com/support/guides/where-to-put-share-buttons-vs-follow-buttons/
* https://www.codeinwp.com/blog/determining-best-placement-social-media-icons/

(disclaimer: I have not idea if we can trust these sources)

Reply:
I can easily [find](https://www.addthis.com/academy/addthis-sharing-and-follow-buttons-best-practices/) people [supporting](https://ux.stackexchange.com/questions/16020/should-social-sharing-buttons-be-at-the-top-or-bottom-of-an-article) each [placement](https://www.quora.com/Conversion-Optimization-What-is-the-best-spot-to-place-Social-Share-buttons-top-or-bottom) of social share widgets on blogs. It seems like a holy war (i.e., vim vs emacs) without defining what the goals are and then A/B testing for those goals.

`¯\_(ツ)_/¯` I'll move it wherever you prefer because I'm not that emotionally invested in the placement; maybe in the future I'll prefer it in the top. In my current personal opinion it looks better at the bottom and also feels more genuine since we can't expect a visitor to share something they haven't yet read. In the end, I feel it is somewhat a vanity widget most people will never click, but I'm happy to be proven wrong.

Reply:
don't get me wrong Chris, me neither :) I just wanted to understand the rational behind the change, I am ok with either.

Reply:
how would you feel about adding optimizely to run a/b test to test different assumptions?

Reply:
Can I pledge to do that in a follow up PR so that we can move faster with these design changes? I ask that because I'm really unhappy with what I currently have for the blog post layout and feel these changes make a big improvement to the appearance and readability of the articles.

Unless you feel strongly that there would be negative consequences, I'd like to prioritize these design changes over optimizations and testing tools, and then iteratively add those features.

Also, how would you feel about using [Google Optimize](https://www.google.com/analytics/optimize/features/) if it is already built into the Google Analytics script we're currently loading?

Reply:
sounds great - I am ok with whatever tool as long as we can run a/b tests with them